### DISCUSSION TITLE
Test domain w/o "suspicious" warning from Phantom x Blowfish

### DISCUSSION THREAD

**Discussion by dimastrauchman** [phantom/docs#164] [Help & Support] [answered]
### Summary

Hey, 

We are using a domain for beta testing with a that's name different from texture.finance. 

Phantom x Blowfish label the test domain as suspicious. Hence show a warning before allowing the user to see Phantom transaction signature interface. 

Is it possible to submit the beta test domain and resolve the warning (domain whitelist or smth)?

Thank you, 

D

### Steps to Reproduce

Go to Texture test domain (can share in DMs w a team member). 

Make any transaction with any of the services.

Our original production domain that's working just fine is https://texture.finance/.  

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by bfriel** [comment, marked as answer]
Hi there, can you please submit the test domain to `review@blowfish.xyz` ? Thank you!
